src/webots/scene_tree/WbAddNodeDialog.cpp

```diff
@@ -229,10 +229,14 @@ void WbAddNodeDialog::updateItemInfo() {
         mInfoText->setPlainText(
           tr("This folder lists all suitable node that were defined (using DEF) above the current line of the Scene Tree."));
         break;
-      case PROTO_EXTRA:
-        mInfoText->setPlainText(tr("This folder lists all suitable PROTO nodes from the extra projects located in: '%1'.")
-                                  .arg(WbPreferences::instance()->value("General/extraProjectsPath").toString()));
-        break;
+      case PROTO_EXTRA: {
+        QString title("This folder lists all suitable PROTO nodes from the preferences Extra project paths and "
+                      "the 'WEBOTS_EXTRA_PROJECT_PATH' environment variable.\n");
```

Comment:
```suggestion
                      "the 'WEBOTS_EXTRA_PROJECT_PATH' environment variable:\n");
```

Reply:
What about leaving the `.` if there's no extra project path defined, and the `:` if there's any? I wouldn't like to leave an open `:` if the list is empty.

Reply:
The "extra project" section is visible in the add node dialog window only if it is non-empty, so there will always be something to show

Reply:
That's a really good point, will address this last comment and upload the changes then